languageserver/integration/multiconfig_integration_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright Flow Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package integration
+
+import (
+	"encoding/json"
+	"net/url"
+	"testing"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-go-sdk"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_CommandsRoutePerClosestFlowJSON(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	loader := &afero.Afero{Fs: fs}
+
+	_ = loader.MkdirAll("/w/a", 0o755)
```

Comment:
Ideally don't ignore errors in tests (and in general). Once the test fails/breaks it becomes very hard to debug. Maybe just wrap in a `require.NoError`